src/results/formulas.jl

```diff
@@ -747,42 +751,132 @@ struct CostOfServiceRebate <: Function
     table_name::Symbol
 end
 function (f::CostOfServiceRebate)(data, table, idxs, yr_idxs, hr_idxs)
-    reg_factor = table.reg_factor
-    return sum0(reg_factor[i] * compute_result(data, f.table_name, :net_total_revenue_prelim, i, yr_idxs, hr_idxs) for i in idxs)
+    reg_factor = table.reg_factor::Vector{Float64}
+    res = 0.0
+    for i in idxs
+        rf = reg_factor[i]
+        rev_prelim = compute_result(data, f.table_name, :net_total_revenue_prelim, i, yr_idxs, hr_idxs)
```

Comment:
I assume this section was changed to speed up this calculation in some way, just checking